help/en/package/jmri/jmrit/dispatcher/NewTrain.shtml

```diff
@@ -163,6 +164,24 @@
           Block identifier to identify the destination Block uniquely.</p>
         </li>
 
+        <li>
+          <a id="trainDetectionOptions"></a> Train Detection Options <span class="since">since 5.7.5</span>
+          <p> This settings effects the ability of auto trains to stop see autotrains for details</p>
```

Comment:
There seems to be a missing dot after "auto train to stop". And that the word "see" should have first letter upper case.

This setting effects the ability of auto trains to stop. See autotrains for details.